src/systems/ec/implementations/vanilla/packages/ec-component-site-header-core/ec-component-site-header-core.scss

```diff
@@ -205,10 +211,21 @@
 
     .ecl-site-header-core__login-toggle,
     .ecl-site-header-core__language-icon {
-      .ecl-icon {
-        height: $ecl-icon-m;
+      .ecl-icon--m {
         margin-right: $ecl-spacing-xs;
```

Comment:
Reading this line and imagining `ecl-site-header-core__language-icon` and `ecl-icon--m` specificity inside, I'm wondering: what about the other icons? What if someone changes the size, then there will be impact on this. Can we avoid it? Maybe both icons can take the `ecl-site-header-core__icon` which hold the size and can be somehow styled this specifically?

Reply:
You are right, better to keep forcing size on these icons. I'm now using a `ecl-site-header-core__icon` class for general headers icons and forcing `m` size with css